Bucketeer/Sources/Public/BKTClient.swift

```diff
@@ -200,6 +238,7 @@ extension BKTClient {
         }
     }
 
+    @available(*, deprecated, message: "use stringEvaluationDetails<T> instead")
```

Comment:
??
```suggestion
    @available(*, deprecated, message: "use getVariationDetail<T> instead")
```

Reply:
hi @kakcy , 

`getVariationDetail` is for internal use only. It can not be accessed from outside.
But I will remote the `<T>` from the `stringEvaluationDetails<T>`

Reply:
I updated it to 
```swift
@available(*, deprecated, message: "use stringVariation(featureId:, defaultValue:) instead")
```